ckanext/canada/commands.py

```diff
@@ -499,8 +499,8 @@ def _trim_package(pkg):
                 'last_modified', 'position']:
             if k in r:
                 del r[k]
-        if r.get('url_type') == 'upload':
-            r['url'] = 'upload'
```

Comment:
os.path isn't the perfect thing to use here because the separator will depend on the operating system. rsplit('/') is fine, or use one of the url splitting functions instead